.changeset/smart-parents-beg.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/integration': patch
+---
+
+Add option to `resolveUrl` that allows for linking to a specific line number when resolving a file URL.
```

Comment:
might rebase this one on master right, both for this and for the function-to-table change

Reply:
Yep these'll disappear automatically